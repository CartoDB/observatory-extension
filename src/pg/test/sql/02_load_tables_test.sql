SET client_min_messages TO WARNING;
\set ECHO none
\echo Loading fixtures...
\i test/fixtures/obs_table.sql
\i test/fixtures/obs_column_table.sql
\i test/fixtures/obs_column.sql
\i test/fixtures/obs_column_to_column.sql
\echo Done.
\unset ECHO
