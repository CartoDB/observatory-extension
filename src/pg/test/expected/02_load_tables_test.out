SET client_min_messages TO WARNING;
\set ECHO none
Loading fixtures...
Done.
